lib/core/network/api/aws_amplify/amplify_service.dart

```diff
@@ -148,7 +148,7 @@ class AmplifyService {
 
       // handle CUSTOM_CHALLENGE challenge
       final loginCode = e.challengeParameters['loginCode'];
-      await eosService.loginWithCode(accountName: accountName, loginCode: loginCode, network: Networks.telos);
+      await eosService.loginWithCode(accountName: accountName, loginCode: loginCode, network: Network.telos);
       print('return challenge $loginCode');
```

Comment:
I guess we can just go over bare usages of constants - telos - and change them all to use the account's network

in a different PR though